storm-core/test/resources/log4j2-test.xml

```diff
@@ -23,7 +23,8 @@
         </Console>
     </Appenders>
     <Loggers>
-        <Logger name="org.apache.zookeeper" level="WARN"/>
+        <Logger name="org.apache.storm.shade.org.apache.zookeeper" level="WARN"/>
+        <Logger name="org.apache.storm.shade.org.apache.curator" level="WARN"/>
```

Comment:
Thanks for finding/fixing this.

Reply:
It got pretty unreadable :) I'll take a look whether we need to do this in other modules as well.